include/revng/ABI/Definition.h

```diff
@@ -145,6 +145,13 @@ type: struct
       ARM ABI), the value of `StackAlignment` should be equal to 4.
     type: uint64_t
 
+  - name: MinimumStackWordSize
```

Comment:
I'd avoid the "word" nomenclature, it's quite overloaded.

Reply:
I went with `MinimumStackArgumentSize`.